Artifacts/MsSql/Data/Security/1070-OrganizationDepartmentMetadata.sql

```diff
@@ -55,16 +72,26 @@ SELECT  @schoolClaimId = ResourceClaimId
 FROM    dbo.ResourceClaims rc 
 WHERE   rc.ClaimName = 'http://ed-fi.org/ods/identity/claims/school'
 
-IF NOT EXISTS (SELECT 1 FROM dbo.ClaimSetResourceClaims csrc INNER JOIN dbo.ClaimSets cs ON csrc.ClaimSet_ClaimSetId = cs.ClaimSetId WHERE cs.ClaimSetName = 'District Hosted SIS Vendor' AND csrc.ResourceClaim_ResourceClaimId = @schoolClaimId)
```

Comment:
dbo.ResourceClaimAuthorizationMetadatas is old table  and new table is ResourceClaimActions  . this line needs to be updated IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaimAuthorizationMetadatas WHERE ResourceClaim_ResourceClaimId = @organizationDepartmentClaimId)

Reply:
INSERT INTO dbo.ResourceClaimAuthorizationMetadatas(ResourceClaim_ResourceClaimId, Action_ActionId, AuthorizationStrategy_AuthorizationStrategyId)  dbo.ResourceClaimAuthorizationMetadatas is old table and new table is ResourceClaimActions .